Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -62,6 +65,7 @@ open class _ChatChannelListVC<ExtraData: ExtraDataTypes>: _ViewController,
     override open func setUp() {
         super.setUp()
 
+        channelsCount = controller.channels.count
```

Comment:
This needs to be after `setDelegate` else it'll always be 0, right?

Reply:
Not really. Here it's just about getting the initial value. Accessing channels loads the local channels lazily.